backend/service/service.go

```diff
@@ -612,3 +624,154 @@ func (s *APIService) UserMeTicketCredentialsGet(ctx context.Context) (openapi.Im
 
 	return openapi.Response(http.StatusOK, MarshalTicketCredentials(tcs)), nil
 }
+
+// EventsPost - Create a new event
+func (s *APIService) EventsPost(ctx context.Context, createEventRequest openapi.CreateEventRequest) (openapi.ImplResponse, error) {
+    logger := log.Ctx(ctx).With().Str("op", "EventsPost").Logger()
+    userID := util.GetUserIDFromContext(ctx)
+    userEmail := util.GetUserEmailFromContext(ctx)
+    if userID == "" || userEmail == "" {
+        return openapi.Response(http.StatusUnauthorized, nil), nil
+    }
+	
+	if createEventRequest.Name == "" {
+		return openapi.Response(http.StatusBadRequest, "Name is required"), nil
+	}
+	if createEventRequest.Description == "" {
+		return openapi.Response(http.StatusBadRequest, "Description is required"), nil
+	}
+	if createEventRequest.Url == "" {
+		return openapi.Response(http.StatusBadRequest, "URL is required"), nil
+	}
+	if createEventRequest.StartDate.IsZero() {
+		return openapi.Response(http.StatusBadRequest, "Start date is required"), nil
+	}
+	if createEventRequest.EndDate.IsZero() {
+		return openapi.Response(http.StatusBadRequest, "End date is required"), nil
+	}
+
+	// create a context registry instance
+    contextRegistry, err := NewContextRegistry(s.contextRegistryAddr, s.ethClient)
+    if err != nil {
+        logger.Err(err).Msg("Failed to create ContextRegistry instance")
+        return openapi.Response(http.StatusInternalServerError, nil), err
+    }
+
+	// create a context string in the format of "[proofpass.io][eventName]"
+	contextString := fmt.Sprintf("[%s][%s]", "proofpass.io", createEventRequest.Name)
```

Comment:
just realized that multiple users could create same event name, let's change this to [proofpass.io][{event_id}]{event_name}

Reply:
also to avoid the usage of magic string, let's create a const on top of this file to store the prefix `[proofpass.io]`